2025-04-04 12:22:46,427 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:22:46,505 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:22:59,437 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:22:59,512 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:23:21,813 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:23:21,904 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:23:48,717 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:23:48,803 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:23:55,032 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:24:14,416 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:24:14,496 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:24:52,699 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:24:52,779 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:24:56,908 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:24:56,995 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:26:55,562 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:26:59,317 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:26:59,408 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:27:08,670 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:27:08,749 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:29:59,711 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:29:59,810 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:30:26,980 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:31:09,999 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:31:44,070 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:34:32,178 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:35:46,008 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:36:01,214 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:36:01,295 INFO: MAPBOX_TOKEN en la ruta: pk.eyJ1IjoiaWF6YXJhdGUxOTk1IiwiYSI6ImNtNjExNzlqMTBqb3MyaW9vamZqODlkZGkifQ.lQ_lfadoe8d_eLqVqVDB [in /Users/macbook/Documents/Centro_Regional/observatorio/app/main/routes.py:112]
2025-04-04 12:37:47,792 INFO: Observatorio startup [in /Users/macbook/Documents/Centro_Regional/observatorio/app/__init__.py:70]
2025-04-04 12:38:03,912 ERROR: Exception on /auth/guest-login [GET] [in /Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 34, in guest_login
    guest = GuestUser()
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 19, in __init__
    self.is_active = True
AttributeError: can't set attribute
2025-04-04 12:38:03,953 ERROR: Exception on /auth/guest-login [GET] [in /Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 34, in guest_login
    guest = GuestUser()
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 19, in __init__
    self.is_active = True
AttributeError: can't set attribute
2025-04-04 12:38:10,508 ERROR: Exception on /auth/guest-login [GET] [in /Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/macbook/Documents/Centro_Regional/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 34, in guest_login
    guest = GuestUser()
  File "/Users/macbook/Documents/Centro_Regional/observatorio/app/auth/routes.py", line 19, in __init__
    self.is_active = True
AttributeError: can't set attribute
